### PR TITLE
Add super heavy plasma gun

### PR DESCRIPTION
**Pull request by r4d6**
## About The Pull Request
Add a Soteria Experimental Plasma Gun, the 'Ragefire'.

It is robust in combat, dealing a wooping 50 burn damage per hit, with an armor penetration of 30.

However, this mighty weapon isn't without flaws, indeed, each shot fired will consume 10u of Industrial Coolant (1 Tungsten, 1 Acetone, 1 Water) out of an attached reagent container. To remove the container, simply use a screwdriver on it.

If the gun ever run out of coolant to use, then it will have a completly random change of exploding upon the next shot, with the same force as a mine !

After all, with great power come great chances of exploding !

### CODE REVIEW

File: code/modules/research/nodes/combat.dm [hunk 1]
```diff
@@ -239,7 +239,8 @@
 							/datum/design/research/item/ammo/heavy_rifle_408,
 							/datum/design/research/item/ammo/laser_mag,
 							/datum/design/research/item/weapon/laser_ak,
-							/datum/design/research/item/weapon/centauri)
+							/datum/design/research/item/weapon/centauri,
+							/datum/design/research/item/weapon/heavy_plasma)
```
Comment (Trilbyspaceclone) [inline code/modules/research/nodes/combat.dm:243]: Leave this out untill you get the sprite

Comment (r4d6) [reply code/modules/research/nodes/combat.dm:243]: It currently got the same sprite as the Purger.

Comment (Trilbyspaceclone) [reply code/modules/research/nodes/combat.dm:243]: untill you get a new sprite for it then add it into rnd

Comment (r4d6) [reply code/modules/research/nodes/combat.dm:243]: Hound is currently working on the sprite.

Comment (Trilbyspaceclone) [reply code/modules/research/nodes/combat.dm:243]: And untill they are done do not have it in rnd ndes

Comment (r4d6) [reply code/modules/research/nodes/combat.dm:243]: I understand why you don't want it, lore and everything, but I am not going to make a commit to remove it from RnD, only to revert it **again** once the sprite is done.
Beside, I doubt @Kazkin will merge this PR before the sprite is done. if he does, you can yell at him.

Comment (r4d6) [reply code/modules/research/nodes/combat.dm]: So I asked Kaz, and it is commented out until the sprite is added....

File: code/modules/research/nodes/combat.dm [hunk 2]
```diff
@@ -239,8 +239,8 @@
 							/datum/design/research/item/ammo/heavy_rifle_408,
 							/datum/design/research/item/ammo/laser_mag,
 							/datum/design/research/item/weapon/laser_ak,
-							/datum/design/research/item/weapon/centauri,
-							/datum/design/research/item/weapon/heavy_plasma)
+							/datum/design/research/item/weapon/centauri
```
Comment (Trilbyspaceclone) [inline code/modules/research/nodes/combat.dm:242]: )

Comment (r4d6) [reply code/modules/research/nodes/combat.dm:242]: Ah

**Comment by Kazkin** [comment]
This ready?

**Comment by r4d6** [comment]
> This ready?

All it need now is Hound's sprite, but otherwise yes.

**Comment by r4d6** [comment]
> This ready?

Yes, the sprite is in.

**Pull request merged**